examples/placegoose/src/db/schema.ts

```diff
@@ -0,0 +1,35 @@
+import { integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+
+export type Gaggle = typeof gaggles.$inferSelect;
+
+export const gaggles = sqliteTable("gaggles", {
+  id: integer("id", { mode: "number" }).primaryKey(),
+  name: text("name").notNull(),
+  territory: text("territory"),
+});
+
+export type Goose = typeof geese.$inferSelect;
+
+export const geese = sqliteTable("geese", {
+  id: integer("id", { mode: "number" }).primaryKey(),
+  gaggleId: integer("gaggle_id", { mode: "number" }).references(
```

Comment:
we might be able to do something with the latest releases of drizzle that allow for automagical camel casing

https://www.youtube.com/watch?v=TIov_V6NeuQ

Reply:
i saw them in that video, will see if there's a link somewhere to the docs on this

Reply:
https://orm.drizzle.team/docs/sql-schema-declaration#camel-and-snake-casing

Reply:
updated

Reply:
sweeee thanks